### DISCUSSION TITLE
IBM PC

### DISCUSSION THREAD

**Discussion by paulmortlock** [sam-ludlow/mame-ao#26] [Q&A]
Does anyone know how to get these running as it ends up saying hard disk C: is required.

![image](https://github.com/sam-ludlow/mame-ao/assets/7123294/c6567980-f372-4be8-87dc-d634b3f9f1f0)

**Comment by sam-ludlow** [comment]
Hello @paulmortlock 

This is a general MAME query really but here's what I know.

Of the PC type machines your best bet is "ct486".

If you can find floppies that boot it will probably work straight away, but not many do.

So you have to set it up like a real PC. I did this a while back and got Window 95 running, and then immediately wondered why I bothered.

Have a look here https://forums.bannister.org/ubbthreads.php?ubb=showflat&Number=86865

Cheers
Sam

**Comment by paulmortlock** [comment]
So the PC titles do not just run you have to do all that do you that is awfully laborious if so.
